src/App.js

```diff
@@ -1,35 +1,30 @@
 import { BrowserRouter, Route, Switch } from "react-router-dom";
-import AddPatient from "./components/dashboard/addPatient/addPatient.component";
-import { NotFound } from "./components/dashboard/navigation/notFound.component";
-import PatientSearch from "./components/dashboard/searchPatient/searchPatient.component";
+import AddPatient from "./components/PatientRegistration/patientRegistration.component";
+import { NotFound } from "./components/navigation/notFound.component";
+import PatientSearch from "./components/dashboard/patientSearch/patientSearch.component";
 import Login from "./components/login./login.component";
+import Navbar from './components/navigation/navbar.component'
 
 import { useEffect, useState } from "react";
 
-// import PatientSearch from "./components/dashboard/searchPatient/searchPatient.component";
-// import { Switch } from "carbon-components-react";
-// import DisplayPatient from "./components/dashboard/displayPatient/displayPatient.component";
-// import AddPatient from "./components/dashboard/addPatient/addPatient.component";
 
 function App() {
   const [isLoggedin, setIsLoggedin] = useState(false);
   const authKey = window.sessionStorage.getItem("auth.credentials");
-  window.addEventListener("storage", () => console.log("storage Changed"));
-  // useEffect(() => {
-  //   setIsLoggedin(authKey);
-  // }, [authKey]);
+  // window.addEventListener("storage", () => console.log("storage Changed"));
```

Comment:
```suggestion
  // window.addEventListener("storage", () => console.log("storage Changed"));
```
remove commented code